editor/filesystem_dock.cpp

```diff
@@ -234,9 +234,11 @@ void FileSystemDock::_update_display_mode() {
 		new_display_mode = DISPLAY_MODE_SPLIT;
 	}
 
-	if (new_display_mode != display_mode || old_display_mode_setting != display_mode_setting) {
-		display_mode = new_display_mode;
-		old_display_mode_setting = display_mode_setting;
+	if (p_force || new_display_mode != display_mode || old_display_mode_setting != display_mode_setting) {
+		if (!p_force) {
```

Comment:
This if(...) is not really needed. If a change happens in the properties but p_force is true, the properties wont be changed until another useless update.